src/customizations/volto/components/manage/Blocks/Listing/withQuerystringResults.jsx

```diff
@@ -5,6 +5,7 @@ CUSTOMIZATIONS:
 - added additional filters
 - added additional fields to pass to @querystring-search (config.settings.querystringAdditionalFields)
 - usedeepCompareEffect and integrate custom logic for searchBlock to make it work with our implementation
+- used [subrequestID] instead [id] of block, as id of subrequest to avoid block unload on duplicate contents with blocks with same id's
```

Comment:
Metti magari il link alla PR del core, così possiamo gestire questo commento nel momento in cui questa non è più una customizzazione ma è così anche in volto